src/program.js

```diff
@@ -616,11 +616,19 @@ Example: $0 --help run.
       },
       'watch-file': {
         describe: 'Reload the extension only when the contents of this' +
-                  'file changes. This is useful if you use a custom' +
+                  ' file changes. This is useful if you use a custom' +
                   ' build process for your extension',
         demandOption: false,
         type: 'string',
       },
+      'watch-ignored': {
+        describe: 'Extension will not reload if anything in this' +
+                  ' directory or file/s changes. This is useful when there' +
+                  ' are a lots of files which are unnecessary to watch' +
+                  ' for example. node_modules folder',
```

Comment:
Nit, how about tweaking the describe message a bit to say something like:

Paths and globs patterns that should not be watched for changes. This is useful if you want to explicitly prevent web-ext from watching part of the extension directory tree, e.g. the node_modules folder.